CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/io/CsvParser.cs

```diff
@@ -28,8 +28,16 @@ public class CsvParser {
         public static T ReadCsvStreamAs<T>(Stream csvStream) {
             return ReadCsvStreamAsJson(csvStream).ToObject<T>();
         }
+        public static JArray ConvertCsvToJArray(Stream stream) {
+            List<List<string>> parsedData = CsvParser.ReadCsvStream(stream);
+            return CsvParser.ConvertToJson(parsedData);
+        }
+
+        public static JArray ConvertCsvToJArray(List<List<string>> parsedCsvData) {
+            return CsvParser.ConvertToJson(parsedCsvData);
+        }
 
-        public static JArray ConvertToJson(List<List<string>> parsedCsvData) {
+        internal static JArray ConvertToJson(List<List<string>> parsedCsvData) {
```

Comment:
To keep cscore backwards compatible for everyone potentially already using this method, here instead of switching it to internal directly, first you have to set it to Obsolete with a note on what new method to use instead, then in eg a few months this method can be switched to internal